index.html

```diff
@@ -188,10 +182,53 @@ <h2>Multi-Post Stories</h2>
                 
                         <button class="see-projects-btn">See Project</button>
                     </div>
-                
+
+                    
                         
                       </div>
 
+                      <section class="contactus-section">
+
+                      
+                    
+                        <h2 class="callme">CONTACT US</h2>
+                
+                        <P class="contact-description">If you have an application you are <br>
+                             interested in developing,
+                             a feature that you <br>
+                             need built or a project that needs coding.
+                              <br> I'd love to help with it </P>
+                    
+                        <form class="form" action="https://formspree.io/f/xlevdeyp" method="post">
+                           
+                                
+                                <input maxlength="30" type="text" class="name" name="user_name" placeholder="Enter name" required>
+                                <input type="email" class="mail" name="user_email" placeholder="Email Address" required>
+                                <textarea maxlength="500" class="textarea" name="user_message" placeholder="Enter text here" required></textarea>
+                                <button class="button" type="submit">Get Intouch</button>
+                                
+                    
+                        </form>
+                
+            
+                    
+                      </section>
+
+                  <footer class="footer">
+
+                    <div class="border"></div>
+                    <div class="icons-section">
+                    
+                        <img src="images/git.svg" alt="">
+                        <img src="images/linkedin.svg" alt="">
+                        <img src="images/othericon.svg" alt="">
+                        <img src="images/twitter.svg" alt="">
+                        <img src="images/micon.svg" alt="">
+                        </div>
+                    <h2>kiokoj81@gmail.com</h2>
+                  </footer>
```

Comment:
- [ ] In addition, please reduce the space between the icons container and the email because currently, it is very big compared to what we have in the Figma design
**HINT** You may consider reducing the padding-bottom of the icons container 😉